<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>248</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="showID">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>님 반갑습니다!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="logoutBtn">
       <property name="text">
        <string>Logout</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>341</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>제품 검색</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>301</width>
       <height>45</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>제품명</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchTxt"/>
      </item>
      <item>
       <widget class="QPushButton" name="searchBtn">
        <property name="text">
         <string>검색</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="searchList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>301</width>
       <height>401</height>
      </rect>
     </property>
     <property name="title">
      <string>검색 결과</string>
     </property>
     <widget class="QListWidget" name="resultList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>281</width>
        <height>391</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QScrollBar:vertical {              
            border: none;
            background:white;
            width:3px;
            margin: 0px 0px 0px 0px;
        }
        QScrollBar::handle:vertical {
            background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
            stop: 0 rgb(32, 47, 130), stop: 0.5 rgb(32, 47, 130), stop:1 rgb(32, 47, 130));
            min-height: 0px;
        }
        QScrollBar::add-line:vertical {
            background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
            stop: 0 rgb(32, 47, 130), stop: 0.5 rgb(32, 47, 130),  stop:1 rgb(32, 47, 130));
            height: 0px;
            subcontrol-position: bottom;
            subcontrol-origin: margin;
        }
        QScrollBar::sub-line:vertical {
            background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
            stop: 0  rgb(32, 47, 130), stop: 0.5 rgb(32, 47, 130),  stop:1 rgb(32, 47, 130));
            height: 0 px;
            subcontrol-position: top;
            subcontrol-origin: margin;
        }</string>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>바코드</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchBTxt"/>
      </item>
      <item>
       <widget class="QPushButton" name="searchBBtn">
        <property name="text">
         <string>검색</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>301</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>바코드 읽기</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>691</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>검색 결과</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>311</width>
       <height>531</height>
      </rect>
     </property>
     <property name="title">
      <string>먹어도 돼요?</string>
     </property>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>291</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>채식주의 정보</string>
      </property>
      <widget class="QLabel" name="resultVImg">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>81</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="resultVText">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>251</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>대체식품</string>
       </property>
       <widget class="QComboBox" name="alterVlist">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>211</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>291</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>알레르기 정보</string>
      </property>
      <widget class="QLabel" name="resultAImg">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>81</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="resultAText">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>170</y>
         <width>251</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>대체식품</string>
       </property>
       <widget class="QComboBox" name="alterAlist">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>211</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>321</width>
       <height>531</height>
      </rect>
     </property>
     <property name="title">
      <string>제품 정보</string>
     </property>
     <widget class="QLabel" name="productImg">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>241</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>281</width>
        <height>231</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>279</width>
         <height>229</height>
        </rect>
       </property>
       <widget class="QLabel" name="productName">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>279</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="productNutrient">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>279</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="productAllergy">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>279</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="productmtrl">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>160</y>
          <width>279</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
